handbook/people-ops/index.md

```diff
@@ -2,7 +2,7 @@
 
 - [Travel](travel.md)
 - [Onboarding](onboarding/index.md)
-- [Distributed work tips](https://github.com/sourcegraph/Graphbook/blob/master/Onboarding/Distributed%20work%20tips.md)
+- [Remote work tips](https://github.com/sourcegraph/Graphbook/blob/master/Onboarding/Distributed%20work%20tips.md)
```

Comment:
Rename the file too

Reply:
@mercadon is moving these files into the handbook anyway very soon so not worth it.

Reply:
Ah, right, I missed that these weren't already in the handbook.